### PR TITLE
Feature/join table

### PR DESCRIPTION
**Pull request by Rana-Mudassir**
##In this pull request, I implement the vet_clinic database by adding the join visit table following these steps:

- [x] CREATE TABLE vets
- [x] CREATE TABLE specializations 
- [x] CREATE TABLE visits 
- [x] INSERT data in vets specializations and visists
- [x] Add queries to retrieve data from tables
- [x] Screenshots attached

1vets

2specializations

3visits

4lastanimal

5animalcount

6allvetswithspeclty

7visitanimalbtwdate

8mostvisttovets

9masyfirstvisit

10detailmostrece

11havenotspecialty

11speciegetmost

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,129 @@
...
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE.md) licensed.
+
```
Comment (Shedrack-Sunday) [inline README.md:128]: - [x] Kindly add a license to the root of your project. That's why your MIT license  is failing

File: data.sql [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+INSERT INTO vets (name, age, date_of_graduation) VALUES('William Tatcher', 45, '2000/4/23'), ('Maisy Smith', 26, '2019/1/17'),
+('Stephanie Mendez', 64, '1981/5/4'), ('Jack Harkness', 38, '2008/6/8');
+
+INSERT INTO specializations (vet_id, species_id) VALUES(
+(SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon')
+)
+
+INSERT INTO specializations (vet_id, species_id) VALUES(
+(SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon')
+)
```
Comment (Shedrack-Sunday) [inline data.sql:65]: - [x] Nice job here. Based on project requirements, you're also meant to do an insertion for `Digimon`. you did only that of `Pokemon`. Kindly add the inersion 

## Sample:
```
INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));
```

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Rana-Mudassir      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use of GitHub features ✔️   
- Nice use of SQL queries 👍 

  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few  features you missed

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 2 above)
Comment (Rana-Mudassir) [reply data.sql:65]: Thanks for the recommendation. 😀
I updated. Screenshot attached.
12updatedspecialty

new

File: README.md (hunk 1 above)
Comment (Rana-Mudassir) [reply README.md:128]: Added License. Thanks for the recommendation 😀

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @Rana-Mudassir 🙋‍♀️

Great job implementing the changes suggested by the previous reviewer 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Well done with SQL queries.
-  ✅ Great work using the right Gitflow 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**